crates/swc_ecma_minifier/src/js.rs

```diff
@@ -215,6 +217,52 @@ pub enum JsMinifyCommentOption {
     PreserveSomeComments,
     #[serde(rename = "all")]
     PreserveAllComments,
+    #[serde(untagged)]
+    PreserveRegexComments(Box<PreserveRegexCommentsInner>),
+}
+
+#[derive(Debug, Clone)]
+pub struct PreserveRegexCommentsInner {
```

Comment:
But if we have serialize/deserialize implementation on `CachedJsRegex` and the original source regex in `CachedJsRegex` this type would be much simpler